README.md

```diff
@@ -1,77 +1,373 @@
 <h1 align="center"><img src="/img/vwc.gif" alt="Vets Who Code" width="100px" /> VETS WHO CODE</h1>
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Web Presence](#web-presence)&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;[Know Your Tool](#command-line)&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;[HTML & CSS](#html)&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;[JavaScript](#javascript)&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;[Capstone](#capstone)
-
 ---
 
 ## [Prework](https://vetswhocode.io)
 
-Welcome to the official Vets Who Code Prework.
-Here you'll find mandatory tutorials that need to be completed to be considered for the program.
-This can also be used as a great refresher for grads and newbies interested alike.
-While, we don't expect you to be experts, but we do expect you to be **familiar**.
-There is a lot of content to consume here, get used to the fire hose! 👨🏻‍🚒
+Welcome to the official Vets Who Code Prework! Here you'll find the tools, work, and accounts to prepare you for Vets Who Code's prior coursework.  We recommend you complete each section as they build upon one another. The prework should take a total of eight hours to complete.
 
-<h2 id="web-presence">Web Presence</h2>
 
-You should have a professional presence on all of the below platforms, as well as a professional email account. A [personal brand](http://blog.thefirehoseproject.com/posts/personal-branding-software-developer/) is really important.
+ ### Accounts
 
-- [Twitter](https://twitter.com)
-- [Codepen](https://codepen.io)
-- [GitHub](https://github.com)
-- [Dev.to](https://dev.to/)
-- [LinkedIn Veteran](https://linkedinforgood.linkedin.com/programs/veterans/premiumform)
+Here are the accounts you will need to store your projects, solve problems, prototype, network, write, and be inspired by others' work. You'll also be able to see some fantastic projects from others.
```

Comment:
Do students need to have these accounts setup? Are we going to insure they're setup in anyway? At the very bottom there is a call out that the code doesn't HAVE to be on Github. Do we need to update the Prework form with urls for Dev.to, Github, and Linkedin?